litmus-portal/graphql-server/pkg/project/create.go

```diff
@@ -34,6 +34,21 @@ func CreateProjectWithUser(ctx context.Context, projectName string, userID strin
 		return nil, err
 	}
 
+	if projectName == "" {
+		return nil, errors.New("project name can't be empty")
+	}
+
+	//checking duplicate projectname
+	filter := bson.D{{"name", projectName}}
```

Comment:
Need to add userid also.

Reply:
adding it

Reply:
@rajdas98 Do we require userid? The project name should be unique across all users.

Reply:
cc: @ajeshbaby